build/generate-svg.js

```diff
@@ -25,10 +25,10 @@ async function processFile(file, {prefix, fullName} = {}) {
     if (prefix === 'octicon') name = name.replace(/-[0-9]+$/, ''); // chop of '-16' on octicons
   }
 
+  // keep the xmlns attribute to make the files could be displayed by local browsers/image-viewers
```

Comment:
```suggestion
  // keep the `xmlns` attribute so that the files are displayble in standalone documents
  // The svg backend module will strip the attribute during startup for inline display
```